### ISSUE TITLE
Smart clip functionality should be accessible for keyboard users

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#2054]
**Describe the bug**
The non-destructive resizing of clips should be accessible for keyboard users. It isn't.

**To Reproduce**
1. Try to non-destructively resize a clip using the keyboard.
2. Fail.

**Expected behavior**
This functionality should be accessible for keyboard users.

**Comment by SteveDaulton** [comment]
I think that non-destructively making a clip smaller is accessible by using "Edit menu > Remove  Special > Trim Audio (Ctrl + T)".
However, expanding a clip looks to be totally inaccessible.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
